src/Skia/Avalonia.Skia/Gpu/OpenGl/GlSkiaGpu.cs

```diff
@@ -69,7 +69,7 @@ public ISkiaSurface TryCreateSurface(PixelSize size)
             catch (Exception e)
             {
                 Logger.TryGet(LogEventLevel.Error, "OpenGL")
-                    ?.Log(this, "Unable to create a Skia-compatible FBO manually");
+                    ?.Log(this, $"Unable to create a Skia-compatible FBO manually Error: {e.Message}.");
```

Comment:
```suggestion
                    ?.Log(this, "Unable to create a Skia-compatible FBO manually Error: {Message}.", e.Message);
```

Reply:
Just curious, is this a performance tip btw? Or is this here just due to `Log` internals?

Reply:
@worldbeater I don't know specific reason for that, but it can be useful for sure for some kinds of logger sinks, that can allow grouping by log-message without parameters and provider some special formatting for these params.
And there could be some performance benefits for sure.